### PR TITLE
[Term Entry] PHP Arrays array_uniqe()

### LINKED ISSUE #3050: [Term Entry] PHP Arrays `array_unique()`
### Reason/inspiration (optional)

We would like a new term entry for [PHP](https://www.codecademy.com/resources/docs/php) about the `array_unique()` function.

The entry should go in a new file in `docs/content/php/concepts/arrays/terms/array-unique/array-unique.md` (Note the dashes rather than underscores.)

The entry should 

- Have a syntax section illustrating the function with a ` ```pseudo ` code block. 
- An example showing the function in use with a ` ```php ` code block. 
- A runnable codebyte example in a ` ```codebyte/php` block. 

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by ChiragAgg5k**
### Description

- Added the term entry for array_uniqe() method in PHP.
- Closes #3050 .

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/php/concepts/arrays/terms/array-unique/array-unique.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
+---
+Title: 'array_unique()'
+Description: 'Used to remove duplicate values from an array.'
```
Comment (caupolicandiaz) [inline content/php/concepts/arrays/terms/array-unique/array-unique.md:3]:
```suggestion
Description: 'Returns an array without duplicate values.'
```

File: content/php/concepts/arrays/terms/array-unique/array-unique.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+Tags:
+  - 'Arrays'
+  - 'Methods'
+  - 'Functions'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The **`array_unique()`** function removes duplicate values from an array.
```
Comment (caupolicandiaz) [inline content/php/concepts/arrays/terms/array-unique/array-unique.md:16]:
```suggestion
The **`array_unique()`** function returns a new array with any duplicate values removed.
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@ChiragAgg5k I've completed my initial review. Please update the entry, and tag me when you've finished making changes. Thanks

**Comment by ChiragAgg5k** [comment]
@caupolicandiaz i have made the changes

**Review by caupolicandiaz — APPROVED** [review]
@ChiragAgg5k Looks good. This is ready for a second review

**Review by SSwiniarski — APPROVED** [review]
@ChiragAgg5k Looks good.

**Pull request merged**